lib/table.rb

```diff
@@ -0,0 +1,63 @@
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+require_relative 'colors.rb'
+
+class Table
+  attr_accessor :table
+
+  def initialize
+    @table = {
+      "1": 1,
+      "2": 2,
+      "3": 3,
+      "4": 4,
+      "5": 5,
+      "6": 6,
+      "7": 7,
+      "8": 8,
+      "9": 9
+    }
+  end
+
+  def show_table(move = nil, sign = nil)
+    @table[:"#{move}"] = case sign
+                         when 'x'
+                           sign.pink
+                         when 'o'
+                           sign.yellow
+                         else
+                           sign
+                         end
+
+    puts "     #{table[:"7"]} | #{table[:"8"]} | #{table[:"9"]}
```

Comment:
Please remove all `puts` statements from files that are in the /lib folder. 
No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))